backend/migrations/versions/2a2f62761a4b_adding_table_details.py

```diff
@@ -20,9 +20,9 @@
 
 def upgrade() -> None:
     op.create_table('details',
-    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('id', sa.BigInteger(), nullable=False),
```

Comment:
Did you edit this file?

Why didn't you create a new migration after changing the Model?  Write out all the steps to migrating a database and what happens during each step and see if you can find why this is the wrong approach.

What did you do to the database to make this work?  What would happen to a real database if you did the same thing?

Reply:
after i made the changes. i saved and ran flask db downgrade base (to clear the database) and then i ran flask db upgrade head to update the changes. 
i would say this would be the wrong approach because if i had a database full of a bunch of information i just got rid of all that information.

Reply:
if i ran a new migration and updated that then i wouldn't be erasing what is already in the database

Reply:
Correct, migrate forward if possible, so I know you know the correct process.